scripts/postdeploy.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+cd app
+
+# Run Django migrations.
+python manage.py migrate
+# Seed database with dummy users
+python manage.py seedusers
```

Comment:
Oh I think I see what you're saying. Yes we can switch back to relying on Heroku's `postdeploy` script as long as we override (on staging and prod) the env var that you're checking for in your script.